app/webpacker/styles/headings.scss

```diff
@@ -158,7 +158,7 @@
 }
 
 .heading--highlight-white-on-purple {
```

Comment:
@garethwilliamsdesign this feels quite confusing if it is still called `highlight-white-on-purple`? does the name need changing? I presume this would mean making other changes elsewhere though